conf/debug_conf_template.toml

```diff
@@ -1,4 +1,5 @@
 title = "Template file containing all configuration for debug"
+cucumberReportPath = "" # Path to directory for the cucuber genereated gherkin test reports
```

Comment:
We discussed that this should not be a global var. Create a specific group for this.